CRM/Case/BAO/Case.php

```diff
@@ -58,6 +58,39 @@ public static function add(&$params) {
     return $result;
   }
 
+  /**
+   * @param \Civi\Core\Event\PostEvent $e
+   */
+  public static function on_hook_civicrm_post(\Civi\Core\Event\PostEvent $e): void {
+    if ($e->action === 'create' && $e->entity === 'Activity') {
```

Comment:
I'll double-check but it's a "feature" that you can **edit** an activity and add `[case #1234]` into the subject and it will file it on case. I should probably make a unit test for that.

Reply:
Yeah editing the subject works on dmaster.demo but not on the test site here.

Reply:
Good catch @demeritcowboy - I've updated it to handle both